### ISSUE TITLE
Mass error for missed monoisotopics is uninformative

### ISSUE THREAD

**Issue by zrolfs** [smith-chem-wisc/MetaMorpheus#1128]
It's difficult to visualize mass error when the monoisotopic error is included in the calculation. For example, a PSM will have a ppm mass error of 600 when really it's a ppm error of 2 after we account for the monoisotopic error. We already have the monoisotopic shift shown in the "notch" column for users to see which ones were errors. It's rather confusing and uninformative to have mass errors far outside the allowed tolerances when using notches.

**Comment by rmillikin** [comment]
I agree; it's a difference between a _selected precursor_ mass error and a _peptide_ mass error. I think most people care about the latter. I'd be OK with exclusively reporting the peptide mass error (i.e., replacing what's currently there instead of tacking on yet another column to the PSM output).

**Comment by trishorts** [comment]
The "Notches" column notes the missed monoisotopic mass (when missed). If it is one, we could add/substract a monoisotopic mass from the expereimental and then report the revised mass error in ppm. Then they would all fall nicely into a single, narrow distribution. We could keep the absolute mass error (Da) to help the user understand that there was a missed monoisotope error.
